implicits/src/com/rallyhealth/weepickle/v1/implicits/MacroImplicits.scala

```diff
@@ -156,13 +163,13 @@ object MacroImplicits {
         index: Int,
         isParamWithDefault: Boolean,
         assumeDefaultNone: Boolean
-      ): Tree = {
+      ): Option[Tree] = {
         val defaultName = TermName("apply$default$" + (index + 1))
-        if (!isParamWithDefault) {
-          if (assumeDefaultNone) q"${TermName("None")}"
-          else q"null"
```

Comment:
This removal is notable. AFAICT, assuming `null` as a default is undefined behavior. It's just a sneaky special case that snuck in somehow (maybe from upickle). Please check me on that @russellremple.

```
@dropDefault case class A(a: String)
FromScala(A(null)).transform(ToJson.string) ==> """{}""" // before
FromScala(A(null)).transform(ToJson.string) ==> """{"a":null}""" // after
```

Note that this has no effect on `Option`s because `ToOption.visitNull` is defined.

Reply:
I think the `q"null"` came from upickle -- I don't believe I invented it (at least I _hope_ I didn't) I don't think it gets used though because the default is only used when `hasDefault` is true, and that only happens if `isParamWithDefault` is true or `isOptionWithoutDefault` is true. So the `q"null"` just sits there unaccessed. However, it is confusing and brittle as currently implemented, and I think keeping the default as an option makes a bunch more sense.